example/code-snippets/main.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 import createClass from 'create-react-class';
 import DateRangePicker from 'react-daterange-picker';
-import moment from 'moment-range';
+import moment from '../moment-range';
```

Comment:
As this is also meant to be an example for an end user to copy, should we unfold this to something similar to the current [src/moment.js](https://github.com/Yaska/react-daterange-picker/blob/a9e0a9ed132db759e1ae8075a9e26617bd9c6c91/src/moment-range.js) implementation? 🤔

Reply:
ah yes, makes sense, let me do that.